src/applications/edu-benefits/complaint-tool/helpers.js

```diff
@@ -73,7 +73,7 @@ function pollStatus(guid, onDone, onError) {
         if (!res || res.data.attributes.state === 'pending') {
           pollStatus(guid, onDone, onError);
         } else if (res.data.attributes.state === 'success') {
-          onDone(res.data.attributes.parsed_response);
+          onDone(res.data.attributes.parsedResponse);
```

Comment:
@jholton will `X-Key-Inflection: camel` convert `parsed_response` to `parsedResponse`?